packages/odyssey/src/scss/components/_field.scss

```diff
@@ -10,26 +10,49 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-.ods-field--hint,
-.ods-field--error {
+/* stylelint-disable no-descending-specificity */
```

Comment:
Can we refactor our stylesheet to be more human friendly and avoid having this lint escape rule? I would like to see the CSS being constructed in a simpler and lint escape free fashion even if it requires being more verbose.